data/na/swagger/1-0-5.yaml

```diff
@@ -2349,6 +2393,142 @@ definitions:
         description: the ID of the payment profile. You need to keep track of this value.
       card:
         $ref: '#/definitions/ProfileCards'
+  RecurringPaymentResponse:
+    properties:
+      account_id:
+        type: number
+        format: integer
+        description: 'Unique account ID'
+      state:
+        type: string
+        maxLength: 6
+        description: 'The state of the Recurring Payment account'
+      language:
+        type: string
+        maxLength: 3
+        description: 'Account's billing language'
+      payment:
+        $ref: '#/definitions/RecurringPaymentDetails'
+      billing:
+        $ref: '#/definitions/Address'
+      card:
+        $ref: '#/definitions/RecurringPaymentCard'
+      custom:
+        $ref: '#/definitions/RecurringPaymentCustom'
+      schedule:
+        $ref: '#/definitions/RecurringPaymentSchedule'    
+  RecurringPaymentDetails:
+    properties:
+      amount:
+        type: number
+        format: double
+        description: 'A decimal value in dollars. Uses up to two decimal places.'
+      disable_tax1:
+        type: boolean
+        description: 'Tax included or not'
+      disable_tax2:
+        type: boolean
+        description: 'Tax included or not'
+      process_back_payments:
+        type: boolean
+        description: 'If a recurring payment is disabled and later reactivated back, payments will be charged if enabled.'
+  RecurringPaymentCard:
+    properties:
+      card_type:
+        type: string
+        maxLength: 2
+        description: 'The type of card used in the transaction. AM = American Express, DI = Diners, JB = JCB, MC = MasterCard, NN = Discover, VI = Visa, PV = Visa Debit, MD = Debit Mastercard'
+        enum: ['AM','DI','JB','MC','NN','VI','PV','MD']
+      name:
+        type: string
+        minLength: 1
+        maxLength: 64
+        description: Card holder name.
+      expiry_year:
+        type: string
+        maxLength: 2
+        description: eg. 25 for 2025.
+      expiry_month:
+        type: string
+        maxLength: 2
+        description: eg. 02 for February.
+  RecurringPaymentCustom:
+    properties:
+      order_number:
+        type: string
+        maxLength: 30
+        description: 'Order number'
+      comments:
+        type: string
+        maxLength: 2000
+        description: alphanumeric
+      ref1:
+        type: string
+        maxLength: 256
+      ref2:
+        type: string
+        maxLength: 256
+      ref3:
+        type: string
+        maxLength: 256
+      ref4:
+        type: string
+        maxLength: 256
+      ref5:
+        type: string
+        maxLength: 256
+  RecurringPaymentSchedule:
+    properties:
+      period:
+        type: string
+        description: 'The billing period is the regular interval at which credit card transactions are charged'
```

Comment:
according to recurring payment API guide "the period is unit of measurement for the frequency of the payment: day, week, month or year. "
I think we should have example of are possible values